suite/tests/api/drdecode_x86.c

```diff
@@ -34,15 +34,17 @@
 
 #include "configure.h"
 #include "dr_api.h"
+#include "tools.h"
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 
 #define GD GLOBAL_DCONTEXT
 
-#define ASSERT(x)                                                                    \
-    ((void)((!(x)) ? (printf("ASSERT FAILURE: %s:%d: %s\n", __FILE__, __LINE__, #x), \
-                      abort(), 0)                                                    \
+#define ASSERT(x)                                                                        \
+    ((void)((!(x)) ? (fprintf(stdout, "ASSERT FAILURE: %s:%d: %s\n", __FILE__, __LINE__, \
```

Comment:
stderr seems better for errors.

Reply:
I agree, I just wanted to preserve the same behavior to avoid "breaking" anything.
The reason I changed it to `fprintf` is that I included `suite/tests/tools.h` to get access to `TESTANY` in `CHECK_CATEGORY`, but `tools.h` has https://github.com/DynamoRIO/dynamorio/blob/e4da1a7711c1f1dc1bb5757445d3e1097837177d/suite/tests/tools.h#L360 which prevents using `printf`.
I'll change it to stderr (similar to other `ASSERT` I've seen: https://github.com/DynamoRIO/dynamorio/blob/e4da1a7711c1f1dc1bb5757445d3e1097837177d/suite/tests/api/ir_x86.c#L71)

Reply:
Oh if tools.h is successfully included then use print()

Reply:
Done. Using `print()` also in this `ASSERT`.